### DISCUSSION TITLE
Nitty-Gritty: Mobx Directed Graph

### DISCUSSION THREAD

**Discussion by SortaASoraous** [mobxjs/mobx#3486] [General]
I know that MobX uses a directed graph of observables and node references, I'm curious, is it possible to get the "evaluation order" of things like which nodes are evaluated first?

Asked in MST but figured I might ask here since this is the actual mobx chat.

Trying to build a visualization that shows the evaluation order of node references :)

**Comment by kubk** [comment]
I am not sure if it helps, but you may try `getDependencyTree`: https://mobx.js.org/understanding-reactivity.html#examples
